hdr_schemata/models/GWDM/1.1/schema.json

```diff
@@ -728,306 +636,208 @@
-                                    },
                                     {
                                           "type": "null"
                                     }
                               ],
-                              "description": "If multiple formats are available please specify. See application, audio, image, message, model, multipart, text, video, https://www.iana.org/assignments/media-types/media-types.xhtml Note: If your file format is not included in the current list of formats, please indicate other. If you are using the HOP you will be directed to a service desk page where you can request your additional format. If not please go to: https://metadata.atlassian.net/servicedesk/customer/portal/4 to request your format.",
-                              "title": "Format"
+                              "description": "Format(s) the dataset can be made available in",
+                              "example": "CSV,JSON,SQL database table",
+                              "title": "Dataset Format"
                         }
                   },
                   "required": [
-                        "vocabularyEncodingScheme",
+                        "vocabularyEncodingSchemes",
                         "conformsTo",
-                        "language",
-                        "format"
+                        "languages",
+                        "formats"
                   ],
                   "title": "FormatAndStandards",
                   "type": "object"
             },
-            "IsPartOfEnum": {
-                  "const": "NOT APPLICABLE",
-                  "title": "IsPartOfEnum"
-            },
-            "Isocountrycode": {
-                  "pattern": "^[A-Z]{2}(-[A-Z]{2,3})?$",
-                  "title": "Isocountrycode",
-                  "type": "string"
+            "Gender": {
+                  "anyOf": [
+                        {
+                              "pattern": "\\b(?:Male|Female|Other)(?:,(?:Male|Female|Other))*\\b",
+                              "type": "string"
+                        },
+                        {
+                              "type": "null"
+                        }
+                  ],
+                  "title": "Gender"
             },
-            "Language": {
+            "Lifestyles": {
                   "anyOf": [
                         {
-                              "$ref": "#/$defs/LanguageEnum"
+                              "pattern": "\\b(?:Smoking|Dietary Habits|Physical Activity|Alcohol)(?:,(?:Smoking|Dietary Habits|Physical Activity|Alcohol))*\\b",
+                              "type": "string"
                         },
                         {
                               "type": "null"
                         }
                   ],
-                  "default": null,
-                  "title": "Language"
+                  "title": "Lifestyles"
             },
-            "LanguageEnum": {
-                  "enum": [
```

Comment:
There are quite a few enums being removed from this file.  Are those fields actually being derestricted or are the definitions just moving elsewhere?

Reply:
I don't know what git is talking about 😂 

it's a new file and it seems to think i've moved a file:
```
[...emata/models/HDRUK/latest/dev/schema.json → hdr_schemata/models/GWDM/1.1/schema.json]
```

I deleted a 'latest' file for HDRUK because it was causing issues with TRASER and it thinks ive moved this to a new GWDM schema.. hence why it thinks there's millions of changes

Reply:
Silly git! Looking again with that in mind I think it all makes sense.